components/account/AccountFollowButton.vue

```diff
@@ -1,10 +1,13 @@
 <script setup lang="ts">
 import type { mastodon } from 'masto'
+import { engagement } from '~~/telemetry/generated/ui'
+import { engagementDetails } from '~~/telemetry/engagementDetails'
 import { toggleFollowAccount, useRelationship } from '~~/composables/masto/relationship'
 
-const { account, command, context, ...props } = defineProps<{
+const { account, command, context, gleanContext, ...props } = defineProps<{
```

Comment:
`gleanContext` would be `profile` in `profile.follow.unfollow`

Several components use `AccountFollowButton`

Reply:
How do you feel about renaming these ones to include something specific to it being the follow button? For instance `profile.follow-btn.unfollow` - I feel like that might make it a little more clear but also may be overkill?

Reply:
I thought "follow" would be sufficient to denote the button :( but your comment is proof that it isn't. I will change all these to `follow-btn`